portfolio/src/main/java/com/google/sps/data/ScheduleRequest.java

```diff
@@ -0,0 +1,69 @@
+package com.google.sps.data;
+
+import java.time.Instant;
+import java.util.Collection;
+/**
+ * Models request that are sent to the algorithm. It can be used for both
+ * creation and import flow.
+ */
+public class ScheduleRequest {
+  private Collection<CalendarEvent> events;
+  private Collection<Task> tasks;
+  private final Instant startTime;
+  private final Instant endTime;
```

Comment:
These times represent the start and end of working hours? If so, maybe a better name for the time variables could be "workHourStartTimeInstant"

Reply:
Done.